src/app/(pages)/home/_components/SponsorButton.tsx

```diff
@@ -0,0 +1,7 @@
+export default function SponsorButton() {
+  return (
+    <button className="bg-blueprint hover:bg-blueprint-400 text-white py-2 px-4 rounded-full w-48 float-right">
```

Comment:
Wrap this button element with an anchor tag that links to the contact page. Maybe consider using LinkButton.tsx in @/components